### PR TITLE
Feature/52 populate

### PR DESCRIPTION
**Pull request by FelixoGudiel**
Added new lines to the populate.json file to simulate a more authentic state for our application. This pouplate takes into account the recent changes in the database. To test this new populate, the new database model must be active. For this, drop the database, create it again, make the migrations (if necessary) and migrate, then run the command `python .\manage.py loaddata populate.json` and verify that all the instances have been created.

### CODE REVIEW

**Review by auroranavas — CHANGES_REQUESTED** [review]
This is such impressive work 🤯. Just a few small details regarding the business logic of our client. For example, the evaluations. If there is an annual evaluation it should not be regarding an exam.

**Comment by FelixoGudiel** [comment]
Thank you for your quick notice! I have taken into account your reccomendations and fixed the populate.json. To be more concrete:
-Lesson_events has been changed to be a clearer example of a excursion or outing.
-New center_exit_authorizations have been added to be coherent with the changes in lesson_events.
-The enum `Expired` is now used in two new entities (a volunteer and a student that doesn't go to any lessons).
-The `is_admin` attribute has been removed from the `user` insertions, even though it did not change the behaviour of the populate.
-A new admin user has been added using the role `ADMIN`

I hope this changes comply with our view of the possible database. If any other uncoherent data is found in the populate, let me know!

File: nexong/admin.py [hunk 1]
```diff
@@ -9,3 +9,10 @@
 admin.site.register(Meeting)
 admin.site.register(Family)
 admin.site.register(Student)
+admin.site.register(Partner)
+admin.site.register(Volunteer)
+admin.site.register(Comment)
```
Comment (auroranavas) [inline nexong/admin.py:14]: Please remove this line and the LessonEvaluation lines since they are not part of the DB anymore.

**Review by auroranavas — CHANGES_REQUESTED** [review]

**Comment by FelixoGudiel** [comment]
You are right, also some errors have appeared because I tried to resolve a merge conflict from the github interface. It now works correctly after the merge with develop.

**Review by auroranavas — APPROVED** [review]
Everything works now. Congratulations on your work!

**Review by manortbla — APPROVED** [review]
Great job, everything is correct.

**Review by pabpercab1 — APPROVED** [review]
Everything works as expected. It was a huge task, and you did a great job!

**Pull request merged**